scala-api/src/test/scala/com/pagerduty/scheduler/SchedulerClientSpec.scala

```diff
@@ -44,6 +47,14 @@ class SchedulerClientSpec extends WordSpecLike with Matchers with ScalaFutures w
     override def send(record: ProducerRecord[String, String]): JFuture[RecordMetadata] = ???
     override def close(): Unit = ???
     override def close(timeout: Long, unit: TimeUnit): Unit = ???
+    override def sendOffsetsToTransaction(
+        offsets: util.Map[TopicPartition, OffsetAndMetadata],
+        consumerGroupId: String
+      ): Unit = ???
+    override def abortTransaction(): Unit = ???
+    override def initTransactions(): Unit = ???
+    override def beginTransaction(): Unit = ???
+    override def commitTransaction(): Unit = ???
```

Comment:
Does adding all these help pass some test?

Reply:
It won't compile otherwise